recipes/jina-hubble-sdk/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "jina-hubble-sdk" %}
+{% set version = "0.2.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/jina-hubble-sdk-{{ version }}.tar.gz
+  sha256: bab9b7b4d8c5965e099e07effd92cc0607b56eead0b4d26d6e5b473cb93daf8f
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.7
+    - setuptools >=18.0
+    - wheel
+    - pip
+  run:
+    - python >=3.7
+    - requests ==2.27.1
+    - aiohttp ==3.8.1
+    - rich >=12.0.0
+
+test:
+  imports:
+    - hubble
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/jina-ai/jina-hubble-sdk/
+  summary: SDK for Hubble API at Jina AI.
+  dev_url: https://github.com/jina-ai/jina-hubble-sdk/
+  license: Apache-2.0
+  license_family: Apache
+  license_file: LICENSE
+
+extra:
+  recipe-maintainers:
+    - mapleeit
+    - nomagick
+    - delgermurun
+    - Andrei997
```

Comment:
I need confirmation from @mapleeit @nomagick @delgermurun and @Andrei997 that they are willing to be listed as maintainers for this recipe.

Reply:
> I need confirmation from @mapleeit @nomagick @delgermurun and @Andrei997 

They are my colleagues.

Reply:
Yes, I can confirm. We're colleagues and maintainers of https://github.com/jina-ai/jina-hubble-sdk.
Thank you.

Reply:
Yes, I can confirm as well.

Reply:
> They are my colleagues.

Because these individuals will be sent invites to the repository and potentially more GitHub notifications we are obligated to make sure everyone approves.

Reply:
Yes. I also confirm. Thank you.

Reply:
Yes. Confirm. Thanks.